### PR TITLE
sprint 7 fix Ui Edit Poll and Logic Edit poll

### PR DESCRIPTION
**Pull request by nhahv0902**

### CODE REVIEW

File: app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollActivity.java [hunk 1]
```diff
@@ -50,27 +62,165 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
     @Override
-    public void initViewPager() {
-        List<Fragment> fragments = new ArrayList<>();
-        fragments.add(EditInforFragment.newInstance(mPoll));
-        fragments.add(EditOptionFragment.newInstance(mPoll));
-        fragments.add(EditSettingFragment.newInstance(mPoll));
-        String[] titles = getResources().getStringArray(R.array.array_edit);
-        mAdapter = new ViewPagerAdapter(getSupportFragmentManager(), fragments, titles);
+    public void submitEditPoll() {
+        switch (mType) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollActivity.java]: use if or comment todo in other case

File: app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollContract.java [hunk 2]
```diff
@@ -7,9 +7,21 @@
  */
 public interface ModifyPollContract {
     interface View extends BaseView {
-        void initViewPager();
+        void previousUI();
+        void nextUI();
+        void showProgress();
+        void hideProgress();
+        void showMessage(String s);
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollContract.java]: s should rename to message or msg

File: app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollActivity.java (hunk 1 above)
Comment (nhahv0902) [inline app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollActivity.java]: fixed

File: app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollContract.java [hunk 3]
```diff
@@ -7,9 +7,21 @@
...
+        void nextUI();
+        void showProgress();
+        void hideProgress();
+        void showMessage(String message);
+        void submitEditPoll();
+        void showMessage(int message);
     }
 
     interface Presenter {
+        void clickSave();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollContract.java]: onClickSave, onClickNext, onClickPrevious

File: app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollPresenter.java [hunk 4]
```diff
@@ -1,12 +1,125 @@
...
+                public void onError(String msg) {
+                    mView.hideProgress();
+                    mView.showMessage(msg);
+                }
+            });
+    }
+
+    @Override
+    public void submitEditOption() {
+        List<Option> listOptionReal = new ArrayList<>();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/fpoll/ui/polledition/ModifyPollPresenter.java]: you should create a new menthod to validate option

**Pull request merged**